custom_components/frigate/views.py

```diff
@@ -216,13 +216,26 @@ def _create_path(self, path: str, **kwargs: Any) -> str | None:
 
         camera = path.split("/")[0]
         if path.endswith("clip.mp4"):
-            return f"clips/{camera}-{event_id}.mp4"
+            return f"api/events/{event_id}/clip.mp4"
         return None
 
     def _permit_request(self, request: web.Request, config_entry: ConfigEntry) -> bool:
         """Determine whether to permit a request."""
         return bool(config_entry.options.get(CONF_NOTIFICATION_PROXY_ENABLE, True))
 
+class VodProxyView(ProxyView):
+    """A proxy for clips."""
```

Comment:
Can we retire the old proxy clips view?

Reply:
Probably yes. Also the recordings proxy view.